src/modules/backupRestore/backupRestoreEntities/entityDefinition.ts

```diff
@@ -11,7 +11,12 @@ export type EntityBackupDefinition<T> = Readonly<{
   displayName: string;
   fetchEntities: (client: ManagementClient) => Promise<T>;
   serializeEntities: (entities: T) => string;
-  addOtherFiles?: (loadedEntities: T, archive: archiver.Archiver, logOptions: LogOptions) => Promise<void>;
+  addOtherFiles?: (
+    loadedEntities: T,
+    archive: archiver.Archiver,
+    logOptions: LogOptions,
+    secureAssetDeliveryKey: string | undefined,
```

Comment:
can you swap the logOptions and secureAssetDeliveryKey? Just for the consistency, I would like to have logOptions as last parameter :)